cli/ssync/main.go

```diff
@@ -42,7 +43,8 @@ func Main() {
 		dstPath := args[0]
 
 		ops := &rest.SyncFileStub{}
-		err := rest.Server(*port, dstPath, ops)
+		ctx, cancelFunc := context.WithCancel(context.Background())
```

Comment:
It seems in this case, I assume there is no need to run cancelFunc to stop the server before the process finished. The change here is just to conform to the interface change.

Reply:
Yes. And this part is no longer used by any longhorn workloads. Now it's only for manually debugging the sync feature.